docker/docker-compose.yml

```diff
@@ -32,15 +35,23 @@ services:
       - POSTGRES_HOST=postgres
       - POSTGRES_DATABASE=automatisch
       - POSTGRES_USERNAME=automatisch_user
+      - POSTGRES_PASSWORD=automatisch_password
+      - ENCRYPTION_KEY=automatisch_encryption_key
+      - APP_SECRET_KEY=automatisch_app_secret_key
+      - WORKER=true
     volumes:
       - automatisch_storage:/automatisch/storage
   postgres:
-    image: 'postgres:14.5'
+    image: 'postgres:14.5'    
     environment:
-      POSTGRES_HOST_AUTH_METHOD: trust
-      POSTGRES_DB: automatisch
-      POSTGRES_USER: automatisch_user
+      - POSTGRES_DB=automatisch
+      - POSTGRES_USER=automatisch_user
+      - POSTGRES_PASSWORD=automatisch_password
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
   redis:
     image: 'redis:7.0.4'
+    volumes:
+      - redis_data:/data
 volumes:
   automatisch_storage:
```

Comment:
Is it possible to keep compose file inside the compose folder? So that we can maintain backward compatibility with the documentation.

Reply:
On second thought, we received feedback in the /r/selfhosted where one suggested keeping the compose file in the root folder as a common practice. We may like to consider this.

Reply:
I would also suggest what Ali said here after a second thought with community feedback. It will break docs (we can update that) but it will be easier for anyone to get docker related files in the root folder.

Reply:
leave Dockerfile and entrypoint.sh in the docker folder and move docker-compose.yml to the root?